app/lib/BaseEditorController.php

```diff
@@ -737,19 +751,29 @@ public function SummaryDisplay($pa_options=null, $pb_ajax_load_displays=FALSE) {
 	 * @param bool $pb_ajax_load_displays Load display using AJAX?
 	 * @return bool
 	 */
-	public function SummaryData($pa_options=null, $pb_ajax_load_displays=FALSE) {
+	public function SummaryData($pa_options=null) {
+		$this->SummaryInfo($pa_options, TRUE);
+
+		$ajax_placement_id = $_GET['va_placement_id'];
+		if (isset($ajax_placement_id)) {
+			$this->view->setVar('ajax_item', $_GET['va_placement_id']);
+		}
+		$this->render('summary_display_html.php');
+	}
+	# -------------------------------------------------------
+	/**
+	 * Generates summary of record data based upon a bundle display for screen (HTML)
+	 *
+	 * @param array $pa_options Array of options passed through to _initView
+	 * @param bool $pb_ajax_load_displays Load display using AJAX?
+	 * @return bool
+	 */
+	public function SummaryInfo($pa_options=null, $pb_load_placement_info=TRUE) {
```

Comment:
Use a named variable here to make it clear what it controls, move Ajax-y logic to where this function is called.